lib/generators/react_on_rails/generator_helper.rb

```diff
@@ -48,3 +56,19 @@ def copy_file_and_missing_parent_directories(source_file, destination_file = nil
     copy_file source_file, destination_file
   end
 end
+
+module GeneratorErrors
```

Comment:
@justin808 @robwise suggested to put this in the GeneratorHelper module. I can move it to a separate file.